src/main/java/gregtech/loaders/recipe/handlers/PipeRecipeHandler.java

```diff
@@ -118,6 +118,15 @@ private static void processPipeTiny(OrePrefix pipePrefix, Material material, IMa
                         'X', new UnificationEntry(OrePrefix.plate, material));
             }
         }
+
+        if (material.hasFluid() && material.getProperty(PropertyKey.FLUID).solidifiesFrom() != null) {
+            RecipeMaps.FLUID_SOLIDFICATION_RECIPES.recipeBuilder()
+                    .notConsumable(MetaItems.SHAPE_MOLD_PIPE_TINY)
+                    .fluidInputs(material.getProperty(PropertyKey.FLUID).solidifiesFrom(L / 2))
+                    .output(pipePrefix, material)
+                    .EUt(VA[ULV]).duration(4)
```

Comment:
This duration should probably be also `material.getMass()`.

And the EUt should use the voltage multiplier, to match with the extruder.